tools/form-generation-tool.adoc

```diff
@@ -0,0 +1,198 @@
+Then *{FORM_GENERATION_TOOL}* locates those JSON Schemas in the project and taking advantage of the Uniforms APIs, processes them and generates static forms as a resource in the project `src/main/resources/forms` folder.
+
+*Generation Prerequisities*
+
+- Make sure your https://incubator.apache.org/projects/kie.html[Apache Kie,window=_blank] project has been compiled, otherwise the CLI won't be able to find all the form schemas.
+
+As we installed the *{FORM_GENERATION_TOOL}* globally, we can run the command from anywhere.
+[source,bash]
+----
+form-generation-tool
+----
+
+This will start a wizard to help you generate the forms:
+
+First, set the path to your https://incubator.apache.org/projects/kie.html[Apache Kie,window=_blank] Project.
+We will use https://github.com/apache/incubator-kie-kogito-examples/tree/stable/kogito-quarkus-examples/process-quarkus-example[process-quarkus-example,window=_blank].
+
+[source,bash]
+----
+Kogito Form Generation CLI
+===========================
+ 
+This tool will help you generate forms for the Processes and User Tasks in your Kogito Projects.
+The tool will search for the User Tasks JSON schemas generated in your project, so make sure the project is build.
+The generated forms will be stored as resources in your project (in src/main/resources/forms folder).
+ 
+? Type your Kogito Project path:---> /path/to/my/projects/process-quarkus-example <---
+----
+
+Select one of the available Form types (Patternfly or Bootstrap 4).
+We will use Bootstrap 4.
+Use arrow keys to change the selection.
+
+[source,bash]
+----
+? Select the Form type:
+  patternfly
+❯ bootstrap
+----
+
+Confirm Selection, type *Y*, to start the Form Generation process.
+
+[source,bash]
+----
+Current selection:
+Project path: /path/to/my/projects/process-quarkus-example
+Form type: bootstrap
+ 
+? Do you want to continue? (Y/n) Y
+----
+
+Once the generation is done, browse to `/path/to/my/projects/process-quarkus-example/src/main/resources/forms` folder.
+
+[source,bash]
+----
+ls /path/to/my/projects/process-quarkus-example/src/main/resources/forms/
+ 
+demo_orderItems.config
+demo_orderItems.html
+demo_orderItems_Verify_order.config
+demo_orderItems_Verify_order.html
+demo_orders.config
+demo_orders.html  
+persons.config
+persons.html
+persons_ChildrenHandling.config
+persons_ChildrenHandling.html
+----
+
+One question that may come to your mind, is why there are so many forms.
+We have only three Workflows in `process-quarkus-example` project. 
+User Tasks also have forms generated.
+Even though there only three Workflows in the process-quarkus-example project, there are more than three HTML files due to the User Task generated forms.
+
+== Form UI
+For Quarkus based https://incubator.apache.org/projects/kie.html[Apache Kie, window=_blank] projects, you can use and test them by using the *jBPM Quarkus DevUI - Deployment* extension.
```

Comment:
@jomarko Here you are refering to the wrong devui artifact, it should be `jBPM Quarkus Dev UI` (without the Deployment), and the dependency is:

```xml
<dependency>
  <groupId>org.jbpm</groupId>
  <artifactId>jbpm-quarkus-devui</artifactId>
</dependency>
```
Quarkus extensions are composed of two mvn artifacts `runtime` and `deployment`. The `runtime` contains the executable code that will be part of the application, while the `deployment` has the code that will be executed during the Quarkus augmentation phase.  In Quarkus apps we must add a dependency to the `runtime` artifact and the `quarkus-maven-plugin` will be in charged of fetching the `deployment` artifact and use it **only** during augmentation.

NOTE: Our `jBPM Quarkus Devui` extension can work adding the `deployment` artifact because it does nothing important during augmentation, but the correct way to set it up is using the `runtime` artifact.